routes/web.php

```diff
@@ -274,65 +274,73 @@
 // API
 Route::group(['as' => 'api.', 'prefix' => 'api', 'namespace' => 'API', 'middleware' => 'auth:api'], function () {
     Route::group(['prefix' => 'v2'], function () {
-        Route::group(['as' => 'chat.', 'prefix' => 'chat', 'namespace' => 'Chat'], function () {
-            Route::post('new', '\App\Http\Controllers\Chat\ChatController@newConversation')->name('new');
-            Route::get('updates', '\App\Http\Controllers\Chat\ChatController@updates')->name('updates');
-            Route::get('presence', '\App\Http\Controllers\Chat\ChatController@presence')->name('presence');
-            Route::group(['as' => 'channels.', 'prefix' => 'channels'], function () {
-                Route::apiResource('{channel_id}/messages', '\App\Http\Controllers\Chat\Channels\MessagesController', ['only' => ['index', 'store']]);
-                Route::put('{channel_id}/users/{user_id}', '\App\Http\Controllers\Chat\ChannelsController@join')->name('join');
-                Route::delete('{channel_id}/users/{user_id}', '\App\Http\Controllers\Chat\ChannelsController@part')->name('part');
-                Route::put('{channel_id}/mark-as-read/{message_id}', '\App\Http\Controllers\Chat\ChannelsController@markAsRead')->name('mark-as-read');
-            });
-            Route::apiResource('channels', '\App\Http\Controllers\Chat\ChannelsController', ['only' => ['index']]);
-        });
+        // No scopes required
+        Route::apiResource('rooms', 'RoomsController', ['only' => ['show']]);
 
-        Route::resource('rooms', 'RoomsController', ['only' => ['show']]);
-
-        Route::group(['prefix' => 'beatmapsets'], function () {
-            Route::get('favourites', 'BeatmapsetsController@favourites');     //  GET /api/v2/beatmapsets/favourites
```

Comment:
Why is this being removed?

Reply:
you can get the same data from the users beatmapsets route, and it seems like that's what the osu! client does already

Reply:
IIRC Lazer is using that endpoint because it's only used for displaying user profiles of other users (i.e. it doesn't do anything with the current users' favourite beatmaps in any way yet).

I'd argue it's worth preserving the favourite beatmaps URL endpoint for convenience sake (maybe moving it to `/me/beatmapsets/favourites`) even if it ends up being an alias to `UsersController@beatmapsets` instead.

Having said that, removing/altering API endpoints should be done in a separate pull request instead of as part of this (unrelated) one.